auth/controller/AuthenticationController.py

```diff
@@ -81,3 +81,9 @@ def getJwtPayload(rawJWT):
         except (jwt.exceptions.DecodeError, sqlalchemy.orm.exc.NoResultFound):
             raise HTTPRequestError(401, "Invalid JWT signaure")
     return jwtPayload
+
+
+def userIdFromJWT(token):
+    if not token:
+        raise HTTPRequestError(401, "not authorized")
+    return getJwtPayload(token[7:])['userid']
```

Comment:
This will only work if the caller supplies a nicely written `bearer ` (space included) won't it?

If we're to ignore the `bearer`part, this should be safer:
```python
return getJwtPayload(token.split(' ')[1])['userid']
```

If not, perhaps use a regexp to filter out the token from the header?
```python
import re
raw_token = re.search('^bearer[ ]+(.*)$', token, flags=re.IGNORECASE).group(1)
return getJwtPayload(raw_token)['userid']
```

Reply:
Done.
I have add a length check to avoid IndexError exception on bearer-less JWT.